### PR TITLE
Show preview window for sync namespace

### PR DESCRIPTION
**Pull request by Cosifne**
Fix https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1731426/
When click sync namespace from solution explorer, pop up the preview change diaglog
image

### CODE REVIEW

**Review by genlu — APPROVED** [review]
:shipit:

**Review by CyrusNajmabadi — APPROVED** [review]
Cool!

**Review by ryzngard — APPROVED** [review]

File: src/VisualStudio/Core/Def/ServicesVSResources.resx [hunk 1]
```diff
@@ -1806,10 +1806,6 @@ Additional information: {1}</value>
     <value>Updating namespaces...</value>
     <comment>"namespaces" is the programming language concept</comment>
   </data>
-  <data name="Namespaces_have_been_updated" xml:space="preserve">
-    <value>Namespaces have been updated.</value>
-    <comment>"Namespaces" is the programming language concept</comment>
```
Comment (jasonmalinowski) [inline src/VisualStudio/Core/Def/ServicesVSResources.resx:1811]: Do we need to keep the localization comment?

**Review by jasonmalinowski — APPROVED** [review]

**Pull request merged**